ui/src/RefreshTokenButton.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="https://apis.google.com/js/platform.js?onload=renderButton" async defer></script>
+
+        <meta name="google-signin-client_id" content="806222273987-jdbngimv3abggnpggkroac2p6r98d8h6.apps.googleusercontent.com">
```

Comment:
Shouldn't this client ID be supplied during build or page render?

Reply:
Yes the client ID should be supplied during build. How do I do that?

Reply:
We can template in the correct value at page render time. For now can you make it `content="$(googleClientId)"` and we can do the substitution when we hook this into Leo (in a separate PR)?